src/ui/testrtc-test.html

```diff
@@ -0,0 +1,180 @@
+<!--
+ *  Copyright (c) 2014 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+-->
+<dom-module id="testrtc-test">
+  <style>
+    .test-output {
+      margin: 0 2em 0 2em;
+      word-wrap: break-word;
+    }
+
+    paper-progress {
+      display: block;
+      min-width: 2em;
+    }
+
+    .header {
+      background-color: transparent;
+      height: 2em;
+      min-width: 80%;
+      display: flex;
+      padding-left: 16px;
+      padding-right: 16px;
+      align-items: center;
+      cursor: pointer;
+    }
+
+    .header, iron-collapse {
+      padding-bottom: 0.5em;
+    }
+
+    .header .title {
+      flex: 1;
+      font-weight: 300;
+      overflow: hidden;
+      text-overflow: ellipsis;
+    }
+
+    :host[state='success'] iron-icon {
+      color: #080;
+    }
+
+    :host[state='failure'] iron-icon {
+      color: #F00;
+    }
+  </style>
+  <template>
+    <div class="header" on-tap="toggle">
+      <span class="title">{{name}}</span>
+      <paper-progress hidden$="[[!_inProgress(progressValue)]]" value="[[progressValue]]"></paper-progress>
+      <iron-icon icon="[[_iconForState(state)]]"></iron-icon>
+    </div>
+    <iron-collapse id="collapse">
+      <template is="dom-repeat" items="[[output]]" as="item">
+        <div class="test-output"><span>[[item]]</span></div>
+      </template>
+    </iron-collapse>
+  </template>
+  <script>
+    var PREFIX_INFO    = '[   INFO ]';
+    var PREFIX_OK      = '[     OK ]';
+    var PREFIX_FAILED  = '[ FAILED ]';
+    var PREFIX_WARNING = '[   WARN ]';
+
+    Polymer({
+      is: 'testrtc-test',
+      properties: {
+        name: { type: String },
+        testFunction: { type: Function },
+        state: {
+          value: "unknown",
+          reflectToAttribute: true
+        },
+        progressValue: { value: null },
+        output: { type: Array, value: function() { return []; } }
```

Comment:
This array does not inherit the appendChild() method used in https://github.com/webrtc/testrtc/blob/polymer1/src/js/main.js#L106.